showcase/src/assets/examples/date-picker/weekdays.html

```diff
@@ -0,0 +1,10 @@
+<stark-date-picker 
+    pickerId="picker-only-weekdays"
+    name="only-weekdays"
+    [date]="currentDate" 
+    [dateFilter]="'OnlyWeekdays'" 
+    label="Choisir une date" 
```

Comment:
I think all our example texts should be in English. What do you think about this ?